src/gui/widgets/EffectView.cpp

```diff
@@ -109,7 +110,9 @@ EffectView::EffectView( Effect * _model, QWidget * _parent ) :
 		{
 			m_subWindow = gui->mainWindow()->addWindowedWidget( m_controlView );
 			m_subWindow->setSizePolicy( QSizePolicy::Fixed, QSizePolicy::Fixed );
-			m_subWindow->setFixedSize( m_subWindow->size() );
+			if (m_subWindow->layout()) {
+				m_subWindow->layout()->setSizeConstraint(QLayout::SetFixedSize);
+			}
```

Comment:
@lukas-w I don't know if it's needed, but isn't it safe to keep previous behavior(probably with `else`)?

Reply:
@PhysSong I don't think it's needed, `setSizePolicy( QSizePolicy::Fixed, QSizePolicy::Fixed )` already does that in the `else` case.